pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -408,6 +410,10 @@ let
       exec "$@"
     '';
 
+  testTargetsString =
+    lib.warnIf (testTarget != "") "haskell-modules/generic-builder.testTarget is deprecated. Use testTargets instead"
+    (lib.concatStringsSep " " testTargets);
```

Comment:
In a follow up change, we could change this to `lib.escapeShellArgs` (which of course causes a rebuild).